src/object.d

```diff
@@ -2914,21 +2973,25 @@ auto byKeyValue(T : V[K], K, V)(T* aa) pure nothrow @nogc
 ///
 @safe unittest
 {
+    import std.stdio : writeln;
+
     auto dict = ["k1": 1, "k2": 2];
-    int sum;
+    int sum = 0;
     foreach (e; dict.byKeyValue)
+    {
+        writeln(e.key, ": ", e.value);
```

Comment:
No writeln in unittests. Use assert. It will replace the call with writeln in the docs.

Maybe `assert(e.key[1] == e.value + '0')`